### PR TITLE
Add support for nanopi boards supported by periph.io

### PR DESCRIPTION
**Pull request by biotinker**
Bee ring is using the nanopi neo air, which is now supported OOTB by periph.io https://github.com/periph/host/pull/30

This means very little is required to use the nanopi neo air with RDK's genericlinux.

Tested this on the real hardware and confirmed that GPIO, i2c, and SPI are all working.

### CODE REVIEW

**Review by penguinland — APPROVED** [review]
LGTM!

As a heads-up: although SPI, I2C, and basic I/O work, digital interrupts will _not_ work via Periph (and therefore most encoders won't work, nor will the ultrasonic sensor). Hardware PWM also won't work, though software PWM should (which is fine at low frequencies, but becomes garbage if you get over 200 Hz).

File: components/board/nanopi/board.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Package nanopi implements a nanopi based board.
```
Comment (randhid) [inline components/board/nanopi/board.go:1]: [nit] tag as experimental please and add supported functionality of: GPIO pin outputs, I2C, SPI, Software PWM, and unsupported functionality of Digital Interrupts (?) and HardwarePWM (sure on this) don't work.

**Review by randhid — APPROVED** [review]
LGTM, one little nit agree with @penguinland Hardware PWM won't work, but if it's not blocking for BeeRing

File: components/board/nanopi/board.go (hunk 1 above)
Comment (biotinker) [reply components/board/nanopi/board.go:1]: Done

**Pull request merged**